pkg/kademlia/peer_discovery.go

```diff
@@ -29,9 +28,6 @@ type peerDiscovery struct {
 	queue discoveryQueue
 }
 
-// ErrMaxRetries is used when a lookup has been retried the max number of times
-var ErrMaxRetries = errs.Class("max retries exceeded for id:")
```

Comment:
Same than before, have you checked if the issues is that the error has not been used where it should be?

Reply:
I'm not sure where this would be used

Reply:
Ok, then fine, it can go away.